### DISCUSSION TITLE
What is the ↖ keystroke in the keybindings

### DISCUSSION THREAD

**Discussion by waveskimmer** [dbeaver/dbeaver#22038] [Q&A]
How does one use this keystroke?   It's definitely not on my keyboard.   

![Keystroke](https://github.com/dbeaver/dbeaver/assets/31574609/8386c7f0-8e89-4f0e-b8b1-ea7936e29061)

**Comment by flambchap** [comment]
This is the `Home` keystroke. It can be achieved by pressing your keyboard's Home button or using the `fn + left arrow` shortcut.
